e2e/npm_translate_lock_subdir_patch/MODULE.bazel

```diff
@@ -11,9 +11,6 @@ npm.npm_translate_lock(
         "//subdir:package.json",
         "//subdir:patches/debug@4.3.4.patch",
     ],
-    patch_args = {
-        "*": ["-p1"],
-    },
```

Comment:
The patch arg has to be `-p1` for this e2e where `pnpm.patchedDependencies` is used, so this exclusion exercises the logic.